service/build.gradle

```diff
@@ -194,9 +194,8 @@ dependencies {
   compile group: 'org.springframework.security', name: 'spring-security-oauth2-resource-server'
   compile group: 'org.springframework.security', name: 'spring-security-oauth2-jose'
   compile group: 'com.nimbusds', name: 'nimbus-jose-jwt', version: '8.17.1' // override to fix vulnerability issue (CVE-2019-17195)
-  compile group: 'com.squareup.okhttp3', name: 'okhttp', version: '4.7.2'
-  compile group: 'com.launchdarkly', name: 'okhttp-eventsource', version: '1.11.0'
```

Comment:
removed both as we have now taken `classifier` from `launchdarkly-java-server-sdk`